onnx/cpp2py_export.cc

```diff
@@ -69,8 +69,13 @@ std::unordered_map<std::string, py::bytes> CallNodeInferenceFunction(
   const auto& inputData = ParseProtoFromBytesMap<const TensorProto>(inputDataByNameBytes);
   const auto& inputSparseData = ParseProtoFromBytesMap<const SparseTensorProto>(inputSparseDataByNameBytes);
 
+  std::unordered_map<std::string, int> opset_imports;
+  opset_imports[schema->domain()] = schema->SinceVersion();
+  shape_inference::GraphInferenceContext graphInferenceContext(valueTypes.second, opset_imports);
```

Comment:
```
  GraphInferenceContext(
      const std::unordered_map<std::string, TypeProto*>& outer_scope_value_types_by_name_in,
      const std::unordered_map<std::string, int> opset_imports_in,
      SymbolTable* symbol_table_in = nullptr,
      const ModelLocalFunctionsMap& model_local_functions_in = {},
      const ISchemaRegistry* schema_registry_in = OpSchemaRegistry::Instance(),
      std::unordered_map<std::string, TensorShapeProto>* generated_shape_data_by_name_in = nullptr,
      const int ir_version_in = IR_VERSION)
```
For now I only passed `outer_scope_value_types_by_name` and `opset_imports`. Not sure whether they are sufficient though. Feel free to let me know if you think other parameters are useful.